src/gui/wizard/owncloudwizard.cpp

```diff
@@ -201,6 +201,11 @@ void OwncloudWizard::slotCurrentPageChanged(int id)
     }
 
     setOption(QWizard::HaveCustomButton1, id == WizardCommon::Page_AdvancedSetup);
+    if (id == WizardCommon::Page_AdvancedSetup && _credentialsPage == _browserCredsPage) {
+        // For OAuth, disable the back button in the Page_AdvancedSetup because we don't want
+        // to re-open the browser.
+        button(QWizard::BackButton)->setEnabled(false);
```

Comment:
Is there the need to re-enable the back button on other pages? In case we add further pages, or OAuth isn't supported and we fall back to http creds?